src/authentication/index.ts

```diff
@@ -4,7 +4,7 @@ import BaseApi from '../baseApi';
 import { objectCheck } from '../utils';
 import { Credentials, TokenResponse } from './types';
 
-const resourceName = '_EMPTY';
+const resourceName = 'deltaSync';
```

Comment:
Why does authentication have resourceName `deltaSync`

Reply:
I set it to `''` now, problem was that when we have stuff like `_EMPTY` in the `Resource` list, it breaks when implementing a test that is supposed to check all resources